sdk/core/src/services/AffinidiVaultEncryptionService.ts

```diff
@@ -0,0 +1,97 @@
+import { KeysService } from '@affinidi/common'
+import { profile } from '@affinidi/tools-common'
+
+import { isW3cCredential } from '../_helpers'
+import { VaultCredential } from '../dto/vault.dto'
+import { IPlatformCryptographyTools } from '../shared/interfaces'
+
+@profile()
+export default class AffinidiVaultEncryptionService {
+  private _keysService
+  private _platformCryptographyTools
+
+  constructor(keysService: KeysService, platformCryptographyTools: IPlatformCryptographyTools) {
+    this._keysService = keysService
+    this._platformCryptographyTools = platformCryptographyTools
+  }
+
+  // TODO: Make this private; it is only used by migration service
```

Comment:
This comment looks outdated as I saw you are using that in `AffinidiVaultStorageService`